donate.html

```diff
@@ -0,0 +1,59 @@
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta property="og:image" content="https://res.cloudinary.com/lesswrong-2-0/image/upload/h_200/v1695841034/lighthaven_logo_ruab6b.png">
+    <meta property="twitter:image:src" content="https://res.cloudinary.com/lesswrong-2-0/image/upload/h_200/v1695841034/lighthaven_logo_ruab6b.png">
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://use.typekit.net/tlf2euo.css">
+    <link rel="shortcut icon" type="image/jpg" href="https://res.cloudinary.com/lesswrong-2-0/image/upload/a_90/v1629334277/Lightcone-02_stkgik.svg"/>
+    <title>Lighthaven</title>
+    <script>
+      var APP_ID = "wtb8z7sj";
+      window.intercomSettings = {
+        app_id: APP_ID
+      };
+    </script>
+    <script>(function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',w.intercomSettings);}else{var d=document;var i=function(){i.c(arguments);};i.q=[];i.c=function(args){i.q.push(args);};w.Intercom=i;var l=function(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/' + APP_ID;var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);};if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})();</script>
+    <script defer src="modal.js"></script>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-NP786MY8K9"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'G-NP786MY8K9');
+    </script>
+  </head>
+<body>
+  <div class="navbar">
+    <a href="index.html" class="logo">
+      <img class="logo-image"
+        src="https://res.cloudinary.com/lesswrong-2-0/image/upload/v1629334277/Lightcone-02_stkgik.svg"
+      />
+      <div class="navbar-title">Lighthaven</div>
+    </a>
+    <div>
+      <a href="https://airtable.com/appIDSB5FN81es9PE/shrkHA9uGBXTexxRN" class="CTA-button-navbar" target="_blank">
+        Submit Event Inquiry
+      </a>
+      <a href="./donate.html" class="CTA-button-navbar" target="_blank">
+        Donate
+      </a>
+    </div>
+  </div>
+  <section class="center-section">
+    <h2 class="title">Donating to Lighthaven</h2>
+    <p>Lighthaven is owned by [company that owns Lighthaven]. We're a nonprofit. We built Lighthaven to help facilitate events that improve humanity's thinking on important problems. </p>
```

Comment:
The company that owns Lighthaven is not a non-profit (it's a bit weird, because as it's owned by a non-profit, it is bound by non-profit stuff, but the company is not a 501c3, which people might expect if we say "We're a nonprofit"). Suggested copy

```
Lighthaven is a project of Lightcone Infrastructure, a project of the Center for Applied Rationality. [...]

You can donate to Lightcone via Paypal. [...]
```

Might be nice to mention our other projects somewhere as well